### PR TITLE
Update migration docs

### PR DESCRIPTION
**Pull request by felixfontein**
Adding a section on how to migrate PRs manually (at least how to adjust some things in them), and a section on how to find content manually.

CC @samccann

### CODE REVIEW

File: docs/collection_migration.md [hunk 1]
```diff
@@ -39,6 +53,31 @@ can either recreate your PR in the new repo, or try to setup the tool on your ow
 
 https://github.com/ansible/prmove
 
+### Moving PRs manually
+
+When moving PRs, you need to know that some things changed. This affects paths where content has to be placed, and Python imports that have to be changed.
```
Comment (samccann) [inline docs/collection_migration.md:58]:
```suggestion
When moving PRs, you need to know that some things have changed. This affects paths where content has to be placed, and Python imports that have to be changed.
```

File: docs/collection_migration.md [hunk 2]
```diff
@@ -39,6 +53,31 @@ can either recreate your PR in the new repo, or try to setup the tool on your ow
...
+- `lib/ansible/module_utils/` → `plugins/module_utils/`
+- `lib/ansible/modules/` → `plugins/modules/`
+- `lib/ansible/plugins/` → `plugins/`
+- `test/units/module_utils/` → `tests/unit/plugins/module_utils/`
+- `test/units/modules/` → `tests/unit/plugins/modules/`
+- `test/units/plugins/` → `tests/unit/plugins/`
+- `test/integration/targets/` → `tests/integration/targets/`
+- `test/sanity/ignore.txt` → `tests/sanity/ignore-2.*.txt`
+
+The next list contains a mapping of Python imports. It assumes that the collection from what something is imported is called `foo.bar`.
```
Comment (samccann) [inline docs/collection_migration.md:71]:
```suggestion
The next list contains a mapping of Python imports. It assumes that the target collection is called `foo.bar`.
```

File: docs/collection_migration.md [hunk 3]
```diff
@@ -39,6 +53,31 @@ can either recreate your PR in the new repo, or try to setup the tool on your ow
...
+
+- `import ansible.module_utils.` → `import ansible_collections.foo.bar.plugins.module_utils.`
+- `import ansible.modules.` → `import ansible_collections.foo.bar.plugins.modules.` (should only happen in unit tests)
+- `import ansible.plugins.` → `import ansible_collections.foo.bar.plugins.` (should only happen in unit tests)
+
+Imports of content not moved from ansible/ansible have to stay as before. Only content moved to collections needs to be imported this way.
```
Comment (samccann) [inline docs/collection_migration.md:77]:
```suggestion
Imports of content still in ansible/ansible stay as before. Only content moved to collections needs to be imported this way.
```

File: docs/collection_migration.md [hunk 4]
```diff
@@ -39,6 +53,31 @@ can either recreate your PR in the new repo, or try to setup the tool on your ow
...
+
+Finally, if your PR does deprecate an option, a feature, a module, or a plugin, there have been slight changes (in that you need to specify the collection name in some cases) as well. The syntax for all kind of deprecations is mentioned [here](https://github.com/ansible-collections/overview/issues/45#issuecomment-645619042). For modules, and in-code deprecations, `ansible-test sanity` will also tell you if you missed something.
```
Comment (samccann) [inline docs/collection_migration.md:79]:
```suggestion
Finally, if your PR does deprecate an option, a feature, a module, or a plugin, there have been slight changes (in that you need to specify the collection name in some cases) as well. The syntax for all kinds of deprecations is mentioned [here](https://github.com/ansible-collections/overview/issues/45#issuecomment-645619042). For modules, and in-code deprecations, `ansible-test sanity` will also tell you if you missed something.
```

File: docs/collection_migration.md (hunk 2 above)
Comment (samccann) [reply docs/collection_migration.md:71]: it's the phrase 'collection from what something is imported' that I find confusing.  So it seems this section is telling me 'if your code used to import from ansible/ansible, it now has to import from the collection where that content moved to.  In this example, the content moved to collection foo.bar'

So the example isn't showing me how to get something from ansible.netcommon  in my community.network collection. Maybe it's worth either adding a note to say some of your old imports may need to import from different collections.

Comment (felixfontein) [reply docs/collection_migration.md]: I completely rewrote this part (1b1fc880d334523370b52e5ab1b5214c3d9fb81d). Is the new version easier to understand?

**Comment by mkrizek** [comment]
@samccann anything else to be done here or is it OK to merge?

**Comment by mkrizek** [comment]
Thanks @felixfontein @samccann !

**Comment by felixfontein** [comment]
@samccann @mkrizek also thanks :)

**Pull request merged**